src/values/TimeValue.js

```diff
@@ -294,13 +294,13 @@ SELF.getCalendarModelTextByUri = function( uri ) {
  * @since 0.7
  */
 SELF.PRECISIONS = [
-	{ id: 'GY', text: 'billion years' },
-	{ id: 'MY100', text: 'hundred million years' },
-	{ id: 'MY10', text: 'ten million years' },
-	{ id: 'MY', text: 'million years' },
-	{ id: 'KY100', text: '100,000 years' },
-	{ id: 'KY10', text: '10,000 years' },
-	{ id: 'KY', text: 'millenium' },
```

Comment:
Are these not used anywhere? Also, is this change consistent with what we have in PHP?

And finally, is this really better? Sure, it's the same format, though is it how those things are normally written? I suspect not, which is supported by http://en.wiktionary.org/wiki/Gyr